include/snowflake/client.h

```diff
@@ -403,6 +419,11 @@ typedef struct SF_TIMESTAMP {
     SF_DB_TYPE ts_type;
 } SF_TIMESTAMP;
 
+/**
+ * Initializes an SF_QUERY_RESPONSE_CAPTURE struct.
+ */
+SF_QUERY_RESULT_CAPTURE sf_query_result_capture_init();
```

Comment:
Can you follow the naming convention for the public methods? Just replace `sf` with `snowflake`

Reply:
Done.